diff --git a/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx b/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx
index d5b8d29..be608d1 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx
@@ -1,6 +1,7 @@
 import React from 'react';
 import { Link } from 'gatsby';
 import styled from '@emotion/styled';
+import { keyframes } from "@emotion/core";
 
 import Image from '@components/Image';
 import { IAuthor } from '@types';
@@ -8,16 +9,17 @@ import { IAuthor } from '@types';
 const Bio: React.FC<IAuthor> = ({ author }) => {
   return (
     <BioContainer>
-      <BioAvatar
+      <BioAvatarContainer
         as={author.authorsPage ? Link : 'div'} 
         to={author.slug}
         data-a11y="false"
         aria-label="Author's bio"
       >
+        <BioAvatar/> 
         <BioAvatarInner>
           <RoundedImage src={author.avatar.medium} />
         </BioAvatarInner>
-      </BioAvatar>
+      </BioAvatarContainer>
       <BioText dangerouslySetInnerHTML={{ __html: author.bio }} />
     </BioContainer>
   );
@@ -28,19 +30,33 @@ export default Bio;
 const BioContainer = styled.div`
   display: flex;
   align-items: center;
-  position: relative;
   left: -10px;
 `;
 
+const BioAvatarContainer = styled.div`
+  position: relative;
+`;
+
+
+const ripple = keyframes`
+  from {
+    transform: scale(1);
+    opacity: 1;
+  }
+
+  to {
+    transform: scale(1.5);
+    opacity: 0;
+  }
+`;
+
 const BioAvatar = styled.div`
   display: block;
-  position: relative;
+  position: absolute;
   height: 40px;
   width: 40px;
   border-radius: 50%;
-  background: rgba(0, 0, 0, 0.25);
-  margin-right: 16px;
-  margin: 10px 26px 10px 10px;
+  animation: 2s ${ripple} infinite;
 
   &::after {
     content: '';
@@ -50,7 +66,7 @@ const BioAvatar = styled.div`
     width: 50px;
     height: 50px;
     border-radius: 50%;
-    border: 1px solid rgba(0, 0, 0, 0.25);
+    border: 2px solid tomato;
   }
 
   &[data-a11y='true']:focus::after {
diff --git a/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx b/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx
index 1df6e45..37f5cb7 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx
@@ -115,12 +115,7 @@ const NavigationHeader: React.FC<{}> = () => {
           aria-label="Navigate back to the homepage"
           back={showBackArrow ? "true" : "false"}
         >
-          {showBackArrow && (
-            <BackArrowIconContainer>
-              <Icons.ChevronLeft fill={fill} />
-            </BackArrowIconContainer>
-          )}
-          <Logo fill={fill} />
+          <Title>S / G</Title>
           <Hidden>Navigate back to the homepage</Hidden>
         </LogoLink>
         <NavControls>
@@ -163,6 +158,10 @@ const BackArrowIconContainer = styled.div`
   `}
 `;
 
+const Title = styled.h1`
+  color:  ${p => p.theme.colors.primary};
+  font-size: 50px;
+`
 const NavContainer = styled.div`
   position: relative;
   z-index: 100;
diff --git a/node_modules/@narative/gatsby-theme-novela/src/gatsby-plugin-theme-ui/prism.ts b/node_modules/@narative/gatsby-theme-novela/src/gatsby-plugin-theme-ui/prism.ts
index 4fb900b..5067ca9 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/gatsby-plugin-theme-ui/prism.ts
+++ b/node_modules/@narative/gatsby-theme-novela/src/gatsby-plugin-theme-ui/prism.ts
@@ -1,8 +1,8 @@
 export default {
   token: `#fff`,
   languageJavascript: `#e8696b`,
-  javascript: `#e8696b`,
-  background: `#292c34`,
+  javascript: `#e6cd69`,
+  background: `#151718`,
   comment: `#5e6a76`,
   string: `#a8e2a8`,
   var: `#b3bac5`,
@@ -10,12 +10,12 @@ export default {
   constant: `#b3bac5`,
   plain: `#fff`,
   doctype: `#e8696b`,
-  tag: `#e8696b`,
-  keyword: `#d49fd4`,
+  tag: `#fd797e`,
+  keyword: `#ffa500`,
   boolean: `#ff5874`,
-  function: `#5F8DC3`,
+  function: `#2fd2f9`,
   parameter: `#F9965D`,
-  className: `#ffcf74`,
+  className: `#ff6347`,
   attrName: `#bf87ba`,
   attrValue: `#a8e2a8`,
   interpolation: `#fff`,
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/.DS_Store b/node_modules/@narative/gatsby-theme-novela/src/sections/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/.DS_Store b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx
index c713aa8..943a3bf 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx
@@ -91,10 +91,6 @@ const SubheadingContainer = styled.div`
   ${mediaqueries.tablet`
     margin-bottom: 60px;
   `};
-
-  ${mediaqueries.phablet`
-    display: none;
-  `};
 `;
 
 const GridControlsContainer = styled.div`
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx
index d833006..7260af9 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx
@@ -92,36 +92,62 @@ const ListItem: React.FC<ArticlesListItemProps> = ({ article, narrow }) => {
 
   const { gridLayout } = useContext(GridLayoutContext);
   const hasOverflow = narrow && article.title.length > 35;
-  const imageSource = narrow ? article.hero.narrow : article.hero.regular;
+  const imageSource = article.isProject ? article.hero.regular : narrow ? article.hero.narrow : article.hero.regular;
   const hasHeroImage =
     imageSource &&
     Object.keys(imageSource).length !== 0 &&
     imageSource.constructor === Object;
+  if(article.isProject) {
+    return (
+      <a target="_blank" href={article.slug} >
+        <Item gridLayout={gridLayout}>
+          <ImageContainer narrow={narrow} gridLayout={gridLayout}>
+            {hasHeroImage ? <Image src={imageSource} /> : <ImagePlaceholder />}
+          </ImageContainer>
+          <div style={{paddingBottom: '10px'}}>
+            <Title dark hasOverflow={hasOverflow} gridLayout={gridLayout}>
+              {article.title}
+            </Title>
+            <Excerpt
+              narrow={narrow}
+              hasOverflow={hasOverflow}
+              gridLayout={gridLayout}
+            >
+              {article.excerpt}
+            </Excerpt>
+          </div>
+        </Item>
+    </a>
+    )
+    
+  }
+  else {
+    return (
+      <ArticleLink to={article.slug} data-a11y="false">
+        <Item gridLayout={gridLayout}>
+          <ImageContainer narrow={narrow} gridLayout={gridLayout}>
+            {hasHeroImage ? <Image src={imageSource} /> : <ImagePlaceholder />}
+          </ImageContainer>
+          <div>
+            <Title dark hasOverflow={hasOverflow} gridLayout={gridLayout}>
+              {article.title}
+            </Title>
+            <Excerpt
+              narrow={narrow}
+              hasOverflow={hasOverflow}
+              gridLayout={gridLayout}
+            >
+              {article.excerpt}
+            </Excerpt>
+            <MetaData>
+              {article.date} · {article.timeToRead} min read
+            </MetaData> 
+          </div>
+        </Item>
+      </ArticleLink>
+    );
+  }
 
-  return (
-    <ArticleLink to={article.slug} data-a11y="false">
-      <Item gridLayout={gridLayout}>
-        <ImageContainer narrow={narrow} gridLayout={gridLayout}>
-          {hasHeroImage ? <Image src={imageSource} /> : <ImagePlaceholder />}
-        </ImageContainer>
-        <div>
-          <Title dark hasOverflow={hasOverflow} gridLayout={gridLayout}>
-            {article.title}
-          </Title>
-          <Excerpt
-            narrow={narrow}
-            hasOverflow={hasOverflow}
-            gridLayout={gridLayout}
-          >
-            {article.excerpt}
-          </Excerpt>
-          <MetaData>
-            {article.date} · {article.timeToRead} min read
-          </MetaData>
-        </div>
-      </Item>
-    </ArticleLink>
-  );
 };
 
 const wide = '1fr';
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx
index e982823..9640211 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx
@@ -10,9 +10,51 @@ interface AuthorArticlesProps {
   articles: IArticle[];
 }
 
-const AuthorArticles: React.FC<AuthorArticlesProps> = ({ articles }) => {
+const articles = [
+  {
+    id: "1",
+    slug: "https://what2binge.netlify.app/",
+    secret: false,
+    title: "What2Binge App",
+    author: "Sadique Galaria",
+    date: "April 30th, 2019",
+    dateForSEO: "2019-04-30T00:00:00.000Z",
+    timeToRead: null,
+    excerpt: "Create a bucket list of all the tv shows and movies you watch and share with your friends!",
+    subscription: true,
+    hero: {
+        regular: {
+          src: 'https://firebasestorage.googleapis.com/v0/b/auth-39cb9.appspot.com/o/what2binge.png?alt=media&token=bc8a1e16-43da-4380-a463-0f7790c0c183'
+        }
+    },
+    isProject: true,
+
+  },
+  {
+    id: "2",
+    slug: "https://expotodo.netlify.app/",
+    secret: false,
+    title: "To-Do App",
+    author: "Sadique Galaria",
+    date: "",
+    dateForSEO: "",
+    timeToRead: null,
+    excerpt: "Personal Todo App made with expo-web",
+    subscription: true,
+    hero: {
+        regular: {
+          src: 'https://firebasestorage.googleapis.com/v0/b/auth-39cb9.appspot.com/o/Screenshot%202020-04-28%20at%208.42.49%20PM.png?alt=media&token=ca0db45b-2eb0-4b65-a0ea-f29307c3093a'
+        }
+    },
+    isProject: true,
+  },
+]
+
+const AuthorArticles: React.FC<AuthorArticlesProps> = () => {
+  console.log(articles);
   return (
     <AuthorArticlesContainer>
+      <Heading>My Projects</Heading>
       <ArticlesList articles={articles} alwaysShowAllDetails />
     </AuthorArticlesContainer>
   );
@@ -40,3 +82,9 @@ const AuthorArticlesContainer = styled.div`
     background: transparent;
   `}
 `;
+
+const Heading = styled.h1`
+    font-size: 40px;
+    margin-bottom: 40px;
+    color: ${p => p.theme.colors.primary};
+`
