diff --git a/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx b/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx
index d5b8d29..570efeb 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/components/Bio/Bio.tsx
@@ -50,7 +50,7 @@ const BioAvatar = styled.div`
     width: 50px;
     height: 50px;
     border-radius: 50%;
-    border: 1px solid rgba(0, 0, 0, 0.25);
+    border: 2px solid tomato;
   }
 
   &[data-a11y='true']:focus::after {
diff --git a/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx b/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx
index 1df6e45..37f5cb7 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/components/Navigation/Navigation.Header.tsx
@@ -115,12 +115,7 @@ const NavigationHeader: React.FC<{}> = () => {
           aria-label="Navigate back to the homepage"
           back={showBackArrow ? "true" : "false"}
         >
-          {showBackArrow && (
-            <BackArrowIconContainer>
-              <Icons.ChevronLeft fill={fill} />
-            </BackArrowIconContainer>
-          )}
-          <Logo fill={fill} />
+          <Title>S / G</Title>
           <Hidden>Navigate back to the homepage</Hidden>
         </LogoLink>
         <NavControls>
@@ -163,6 +158,10 @@ const BackArrowIconContainer = styled.div`
   `}
 `;
 
+const Title = styled.h1`
+  color:  ${p => p.theme.colors.primary};
+  font-size: 50px;
+`
 const NavContainer = styled.div`
   position: relative;
   z-index: 100;
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/.DS_Store b/node_modules/@narative/gatsby-theme-novela/src/sections/.DS_Store
new file mode 100644
index 0000000..4765b3c
Binary files /dev/null and b/node_modules/@narative/gatsby-theme-novela/src/sections/.DS_Store differ
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/.DS_Store b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/.DS_Store
new file mode 100644
index 0000000..5008ddf
Binary files /dev/null and b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/.DS_Store differ
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx
index c713aa8..943a3bf 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.Hero.tsx
@@ -91,10 +91,6 @@ const SubheadingContainer = styled.div`
   ${mediaqueries.tablet`
     margin-bottom: 60px;
   `};
-
-  ${mediaqueries.phablet`
-    display: none;
-  `};
 `;
 
 const GridControlsContainer = styled.div`
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx
index d833006..891bc8e 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx
@@ -92,36 +92,62 @@ const ListItem: React.FC<ArticlesListItemProps> = ({ article, narrow }) => {
 
   const { gridLayout } = useContext(GridLayoutContext);
   const hasOverflow = narrow && article.title.length > 35;
-  const imageSource = narrow ? article.hero.narrow : article.hero.regular;
+  const imageSource = article.isProject ? article.hero.regular : narrow ? article.hero.narrow : article.hero.regular;
   const hasHeroImage =
     imageSource &&
     Object.keys(imageSource).length !== 0 &&
     imageSource.constructor === Object;
+  if(article.isProject) {
+    return (
+      <a target="_blank" href={article.slug} >
+        <Item gridLayout={gridLayout}>
+          <ImageContainer narrow={narrow} gridLayout={gridLayout}>
+            {hasHeroImage ? <Image src={imageSource} /> : <ImagePlaceholder />}
+          </ImageContainer>
+          <div style={{paddingBottom: '10px'}}>
+            <Title dark hasOverflow={hasOverflow} gridLayout={gridLayout}>
+              {article.title}
+            </Title>
+            <Excerpt
+              narrow={narrow}
+              hasOverflow={hasOverflow}
+              gridLayout={gridLayout}
+            >
+              {article.excerpt}
+            </Excerpt>
+          </div>
+        </Item>
+    </a>
+    )
+    
+  }
+  else {
+    return (
+      <ArticleLink to={article.slug} data-a11y="false">
+        <Item gridLayout={gridLayout}>
+          <ImageContainer narrow={narrow} gridLayout={gridLayout}>
+            {hasHeroImage ? <Image src={imageSource} /> : <ImagePlaceholder />}
+          </ImageContainer>
+          <div>
+            <Title dark hasOverflow={hasOverflow} gridLayout={gridLayout}>
+              {article.title}
+            </Title>
+            <Excerpt
+              narrow={narrow}
+              hasOverflow={hasOverflow}
+              gridLayout={gridLayout}
+            >
+              {article.excerpt}
+            </Excerpt>
+            <MetaData>
+              {article.date} · {article.timeToRead} min read
+            </MetaData> 
+          </div>
+        </Item>
+      </ArticleLink>
+    );
+  }
   
-  return (
-    <ArticleLink to={article.slug} data-a11y="false">
-      <Item gridLayout={gridLayout}>
-        <ImageContainer narrow={narrow} gridLayout={gridLayout}>
-          {hasHeroImage ? <Image src={imageSource} /> : <ImagePlaceholder />}
-        </ImageContainer>
-        <div>
-          <Title dark hasOverflow={hasOverflow} gridLayout={gridLayout}>
-            {article.title}
-          </Title>
-          <Excerpt
-            narrow={narrow}
-            hasOverflow={hasOverflow}
-            gridLayout={gridLayout}
-          >
-            {article.excerpt}
-          </Excerpt>
-          <MetaData>
-            {article.date} · {article.timeToRead} min read
-          </MetaData>
-        </div>
-      </Item>
-    </ArticleLink>
-  );
 };
 
 const wide = '1fr';
diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx
index e982823..661e745 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/author/Author.Articles.tsx
@@ -10,7 +10,67 @@ interface AuthorArticlesProps {
   articles: IArticle[];
 }
 
-const AuthorArticles: React.FC<AuthorArticlesProps> = ({ articles }) => {
+const articles = [
+  {
+    id: "1",
+    slug: "https://sadiquegalaria.netlify.com",
+    secret: false,
+    title: "My Portfolio",
+    author: "Sadique Galaria",
+    date: "April 30th, 2019",
+    dateForSEO: "2019-04-30T00:00:00.000Z",
+    timeToRead: null,
+    excerpt: "My Old Portfolio Website",
+    subscription: true,
+    hero: {
+        regular: {
+          src: 'https://i.ytimg.com/vi/gtJ5i6DhbaM/maxresdefault.jpg'
+        }
+    },
+    isProject: true,
+
+  },
+  {
+    id: "2",
+    slug: "https://sadique523.github.io/weather/",
+    secret: false,
+    title: "Weather App",
+    author: "Sadique Galaria",
+    date: "",
+    dateForSEO: "",
+    timeToRead: null,
+    excerpt: "Weather App built with weater.api",
+    subscription: true,
+    hero: {
+        regular: {
+          src: 'https://wallpaperstock.net/wallpapers/thumbs1/40194wide.jpg'
+        }
+    },
+    isProject: true,
+  },
+  {
+    id: "3",
+    slug: "https://sadique523.github.io/news/",
+    secret: false,
+    title: "News App",
+    author: "Sadique Galaria",
+    date: "",
+    dateForSEO: "",
+    timeToRead: null,
+    excerpt: "News App buit with some api",
+    subscription: true,
+    hero: {
+        regular: {
+          src: 'https://kingfisher.scene7.com/is/image/Kingfisher/4000776856012_01c'
+        }
+    },
+    isProject: true,
+  },
+ 
+]
+
+const AuthorArticles: React.FC<AuthorArticlesProps> = () => {
+  console.log(articles);
   return (
     <AuthorArticlesContainer>
       <ArticlesList articles={articles} alwaysShowAllDetails />
